streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -289,6 +289,16 @@ public void testCannotCleanupWhileRunning() throws Exception {
         }
     }
 
+    @Test
+    public void testToString()
+    {
+        streams.start();
+        String streamString = streams.toString();
+        streams.close();
+        Assert.assertNotEquals("streamString should not be empty", "", streamString);
+        Assert.assertNotNull("streamString should not be null", streamString);
```

Comment:
Thanks for the review. I added to check that appId is not null or non-empty in `streamsString `